src/parsers/manifestjson.js

```diff
@@ -395,6 +413,33 @@ export default class ManifestJSONParser extends JSONParser {
       messages.MANIFEST_FIELD_UNSUPPORTED,
     ];
 
+    // Message with the following codes will be:
+    //
+    // - omitted if the add-on is being explicitly validated as privileged
+    //   (when --privileged-addon cli option was passed or privilegedAddon
+    //   linter config is being set to true, e.g. locally as part of a CI
+    //   job or when validated as part of submitting the addon to the service
+    //   that will sign it as privileged).
```

Comment:
Where is this done? The code below only considers `isAlreadySigned` as far as I can see.